components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -24,6 +24,7 @@ import {
 } from 'utils/constants';
 
 import AnnouncementBar from '../default_announcement_bar';
+import withGetCloudSubscription from '../../common/hocs/with_get_cloud_subcription';
 
 type Props = {
     userLimit: number;
```

Comment:
Hi Maria. I think it's still needed since the HOC actually just gets passed these props. IMO, it's also better to leave it there so that we're aware that it's possible to pass that kind of prop to this component even without the HOC.

Reply:
Makes sense, good call!